src/docs/cookbook/networking/background-parsing.md

```diff
@@ -43,7 +43,8 @@ dependencies:
 
 ## 2. Make a network request
 
-In this example, fetch a JSON large document that contains a list of
+In this example,
+you'll fetch a JSON large document that contains a list of
```

Comment:
Do we want to use the future tense here? I'm curious because we've specifically introduced it here. 
https://developers.google.com/style/tense

![Screen Shot 2020-03-10 at 4 35 49 PM](https://user-images.githubusercontent.com/5550175/76368363-3ab60500-62ed-11ea-9d78-6d07458048dd.png)

Reply:
I'm ok with future tense at the beginning of a doc, so long as it switches to present as you walk through it.